apps/spruce/src/components/PatchesPage/StatusSelector.tsx

```diff
@@ -56,11 +55,6 @@ const treeData = [
     value: PatchStatus.Started,
     key: PatchStatus.Started,
   },
-  {
```

Comment:
Should we replace this with `PatchStatus.Success` to allow filtering on success?

Reply:
Good catch